### PR TITLE
Operators

### PR DESCRIPTION
**Pull request by Entomy**
Adds back classification of operators, with specific designations for groups, if themes get specific. `:=` has also been classified as `keyword.operator.new.ada` instead of `punctuation.ada`, since it's much closer in semantics to an operator than a delimiter.

### CODE REVIEW

File: integration/vscode/ada/syntaxes/ada.tmLanguage.json [hunk 1]
```diff
@@ -958,6 +1021,10 @@
 				{ "include": "#expression" }
 			]
 		},
+		"multiplying_operator": {
+			"name": "keyword.operator.multiplying.ada",
+			"match": "(?i)\\b(\\*|/|mod|rem)\\b"
```
Comment (gcluzel) [inline integration/vscode/ada/syntaxes/ada.tmLanguage.json:1026]: I have this.
image

Replace with:
```
"(?i)(\\*|/|\\bmod\\b|\\brem\\b)"
```

File: integration/vscode/ada/syntaxes/ada.tmLanguage.json [hunk 2]
```diff
@@ -832,6 +891,10 @@
 				{}
 			]
 		},
+		"highest_precedence_operator": {
+			"name": "keyword.operator.highest-precedence.ada",
+			"match": "(?i)\\b(\\*\\*|abs|not)\\b"
```
Comment (gcluzel) [inline integration/vscode/ada/syntaxes/ada.tmLanguage.json:896]:
```
"(?i)(\\*\\*|\\babs\\b|\\bnot\\b)"
```

**Comment by setton** [comment]
Argh, my merging #414 has introduced conflicts. Let me know if you'd like me to sort them out on this branch.

**Comment by Entomy** [comment]
Really? There shouldn't be. Git must be incorrectly determining changes and thinking there is a conflict. I'll resolve it later.

**Comment by Entomy** [comment]
I have a surprise double... again.

If you wanna tackle this go ahead. Otherwise I'll get to it whenever I finally can.

**Comment by setton** [comment]
Okay, I have used the online editor to resolve the merge, I think I succeeded, but a reread would be appreciated!

**Comment by Entomy** [comment]
Looks good to me

**Comment by setton** [comment]
(swearing in my beard) c'mon github you can do this

**Comment by setton** [comment]
ok I had to do a separate branch and PR (#428) to put your changes and merge manually once more...

**Comment by Entomy** [comment]
I really hate git sometimes. But alright, thanks!

**Pull request closed without merging**